index.html

```diff
@@ -10,29 +10,53 @@
       href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
 
     <title>My Portfolio</title>
-    <link rel="stylesheet" type="text/css" href="style.css" />
   </head>
   <body>
     <header class="header-section">
+      <div class="mobile-menu">
+        <img
+          src="Images/ClosingIcon.png"
+          class="closing-icon"
+          alt="closed icon"
+        />
+        <ul class="right-sidebar">
+          <li class="menu-li">
+            <a href="#portfolio" class="nav-link">Portfolio</a>
+          </li>
+          <li class="menu-li"><a href="#about" class="nav-link">About</a></li>
+          <li class="menu-li">
+            <a href="#contact" class="nav-link">Contact</a>
+          </li>
+        </ul>
+      </div>
+
       <a class="header-logo" href="#">
         <div>
           <h2 class="my-logo">My Logo</h2>
         </div>
       </a>
-
-      <nav class="mobile-menu">
-        <div class="navigation-right">
-          <img src="Images/Enabled.png" alt="mobile menu" />
+      <nav>
+        <div class="nav-right">
+          <img src="images/Enabled.png" alt="normal button" />
```

Comment:
- [ ] Kindly fix the image path because the image directory starts with the Capital `I`.